calico/felix/actor.py

```diff
@@ -17,7 +17,85 @@
 felix.actor
 ~~~~~~~~~~~
 
-Actor infrastructure used in Felix.
+A queue-based Actor framework that supports efficient handling of
```

Comment:
All this commenting seemed to get lost in the big gevent-proto->etcd merge but I think that was all that was lost.